activeresource.gemspec

```diff
@@ -21,8 +21,8 @@ Gem::Specification.new do |s|
 
   s.required_ruby_version = '>= 2.2.2'
 
-  s.add_dependency('activesupport', '~> 5.x')
-  s.add_dependency('activemodel',   '~> 5.x')
+  s.add_dependency('activesupport', '< 5.1')
```

Comment:
Adding `gem 'rails-observers', github: 'rails/rails-observers'` to the Gemfile makes sense since the only supporting version of rails-observers is the master branch.